### ISSUE TITLE
Figure out a better icon for icon-less raid modslots

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#9791]
Bungie stopped drawing nice icons into the raid mod slot and now we show empty circles:

<img width="339" alt="Screenshot 2023-08-28 at 6 42 40 PM" src="https://github.com/DestinyItemManager/DIM/assets/313208/cef5e680-46e9-4284-96b2-e319bd38e889">

We should probably try something else - perhaps we could put the icon overlay here, with the assumption that only one raid will be released per season?

**Comment by robojumper** [comment]
Alternatively we could grab a nice icon from the defs, e.g. https://data.destinysets.com/i/Milestone:292102995

**Comment by bhollis** [comment]
Yeah that'd be great, we'd probably need to make yet another d2ai mapping for it though.

**Comment by robojumper** [comment]
Mod slot metadata is massively hardcoded in DIM already, shouldn't be too hard to add one more def hash to the rest of the hashes we add every 6 months
